crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -104,6 +107,13 @@ impl Settlement {
     }
 }
 
+#[derive(Debug)]
+pub struct ErrorWithAuction {
+    #[allow(dead_code)]
+    pub inner: Error,
```

Comment:
why does inner need to be public? We could keep the struct private and thus be more free with the naming (pub Error, private Inner).

Reply:
resolved